static/docs/user-guide/contributing-documentation.md

```diff
@@ -97,13 +97,20 @@ setting up.
   configured. You may also run `npx prettier --write <file path(s)>` manually
   before committing changes.
 
-- **Markdown**: Using `dvc <command>` in the documentation will create a link to
-  that command automatically. No need to use `[]()` explicitly to create them.
+- Using `dvc <command>` in the documentation will create a link to that command
+  automatically. No need to use `[]()` explicitly to create them.
 
-- **Markdown**: Syntax highlighting in fenced code blocks should use the `usage`
-  and `dvc` custom languages. `usage` is employed to show `dvc help` commands
+- Bullet lists shouldn't be too long, nor each bullet's text (3 sentence
+  paragraphs max.) The bullets can begin in lower case and have no ending
+  punctuation if they're not full sentences (otherwise end with period `.`). The
```

Comment:
`otherwise end with period `.` and start with capital letter` ?

Reply:
Clarified in b866d1b